### PR TITLE
Rewrote the user guide section about the settings dialog

### LINKED ISSUE #14927: Give information on changing settings categories in NVDA's settings dialog
### Is your feature request related to a problem? Please describe.

NVDA's settings dialog is divided into categories.  The user guide covers these in section 12: https://www.nvaccess.org/files/nvda/documentation/userGuide.html#NVDASettings

In 12.1, the guide mentions there are shortcut keys to jump to specific categories and also talks about changing categories.  It would be worth including a sentence clarifying that you can change categories with control+tab / control+shift+tab.

### Describe the solution you'd like

The last line before of 12.1 reads:
The settings categories found in the NVDA Settings dialog will be outlined below.

A sentence could be added after this reading:

To change between categories, press control+tab to move forward, or control+shift+tab to move backwards, or use tab or shift+tab to move to "Categories" list and use the arrow keys.

### Describe alternatives you've considered

Although these are standard Windows controls, and thus not otherwise covered in the user guide, as it covers navigating a specific dialog within NVDA, it doesn't hurt to confirm this is how the dialog works as there can be alternate implementations of similar controls.  We have recieved a query from a user on this.

(These keys are currently covered in the Basic Training for NVDA module as well, though not in the context of the NVDA settings dialog).

### PR DESCRIPTION
**Pull request by XLTechie**
### Link to issue number:

Fixes #14927 

### Summary of the issue:

The issue asked that Ctrl+Tab and Shift+Ctrl+Tab be listed in the user guide section regarding the settings dialog, since many users don't know they can use those to move between categories.

### Description of user facing changes

While editing the section, I found some awkward and seemingly redundant wording, and some unclear portions.

* Reworded the initial paragraph to be more clear about categories and settings.
* Added a note that Ctrl+Tab and Shift+Ctrl+Tab can move between settings categories.
* Clarified a sentence about what shortcut keys for categories do.
* Made the note about adding shortcut keys for categories more clear.  As it was written previously, it implied that adding shortcut keys might be the ONLY way to access those categories.

### Description of development approach

I tried to imagine myself in the position of a user who didn't know much about manipulating settings or their dialogs; and rewrote a bit from that prospective.

### Testing strategy:

The test will be whether @Qchristensen still talks to me after reading this.

### Change log entries:

Changes
Clarified the user guide section describing the settings dialog.

### Code Review Checklist:

There is no code.

### CODE REVIEW

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -1296,16 +1296,19 @@ Some settings can also be changed using shortcut keys, which are listed where re
...
 %kc:settingsSection: || Name | Desktop key | Laptop key | Description |
-The NVDA Settings dialog contains many configuration parameters that can be changed.
-This dialog contains a list with several categories of settings to choose from.
-When you select a category, several settings related to this category will be shown in this dialog.
-These settings can be applied using the apply button, in which case the dialog will stay open.
+NVDA provides many configuration parameters that can be changed using the settings dialog.
+To make it easier to find the kind of settings you want to change, the dialog displays a list with several configuration categories to choose from.
+When you select a category, all of the settings related to it will be shown in the dialog.
+To move between categories, use ``tab`` or ``shift+tab`` to reach the list of categories, and then use the up and down arrow keys to navigate the list.
+From anywhere in the dialog., you may also move forward one category by pressing ``ctrl+tab``, or back one category by pressing ``shift+ctrl+tab``.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:1303]: Fix up punctuation

```suggestion
From anywhere in the dialog, you may also move forward one category by pressing ``ctrl+tab``, or back one category by pressing ``shift+ctrl+tab``.
```

**Review by seanbudd — COMMENTED** [review]
Generally looks good to me, thanks @XLTechie

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -1296,16 +1296,19 @@ Some settings can also be changed using shortcut keys, which are listed where re
 
 ++ NVDA Settings ++[NVDASettings]
 %kc:settingsSection: || Name | Desktop key | Laptop key | Description |
-The NVDA Settings dialog contains many configuration parameters that can be changed.
-This dialog contains a list with several categories of settings to choose from.
-When you select a category, several settings related to this category will be shown in this dialog.
-These settings can be applied using the apply button, in which case the dialog will stay open.
+NVDA provides many configuration parameters that can be changed using the settings dialog.
+To make it easier to find the kind of settings you want to change, the dialog displays a list with several configuration categories to choose from.
```
Comment (Qchristensen) [inline user_docs/en/userGuide.t2t:1300]:
```suggestion
To make it easier to find the kind of settings you want to change, the dialog displays a list of configuration categories to choose from.
```

The list contains 15 items now, not counting any extras added by add-ons.  That seems like more than "several" categories

**Review by Qchristensen — CHANGES_REQUESTED** [review]
Just one small suggestion (Change "a list with several configuration categories", to "a list of configuration categories" - since there are more than "several" now.  Otherwise, all good.

**Review by seanbudd — APPROVED** [review]

**Pull request merged**